pkg/providers/equinix.go

```diff
@@ -34,7 +34,7 @@ func EquinixProviderFactory(providerInfo string, secretData map[string][]byte) (
 	config := equinixProviderConfig{
 		ProjectID: "",
 		Token:     "",
-		Facility:  []string{"da11", "dc10", "dc13", "ny5", "sv15"},
+		Metro:     "da",
```

Comment:
Previous locations were taken from the Dallas / New York / Silicon Valley (metros). Is it possible to specify a _list_ of metros? Such to avoid concentrating all the requests / availability in a single metro only (even thought the Dallas one has eight dc)